waspc/waspls/src/Wasp/LSP/Debouncer.hs

```diff
@@ -86,13 +86,13 @@ debounce (Debouncer running) waitMicros event fire = do
 -- has its own debounce timer. Construct a debouncer with 'newDebouncerIO'.
 --
 -- See 'debounce' for how to use it.
-newtype Debouncer k = Debouncer
-  { -- | A thread-safe map of event labels to async actions. This stores
-    -- actions for each event that are waiting for their debounce timers to end.
-    -- This is needed so that new calls to 'debounce' can cancel the currently
-    -- running actions before they can finish running.
-    debouncerRunningEvents :: STM.Map k (Async ())
-  }
+newtype Debouncer k = Debouncer (DebouncerRunningEvents k)
```

Comment:
Ok that is one solution! I was suggesting the opposite though, calling it differently, not rewriting its definition. Field probably existed for a purpose, which was that it is a nice way to document what it is, that value. And this weird with the extra `type` is a bit unusual, not sure that it makes sense.

Reply:
Ahh re-read the original comment and got it now, one sec

Reply:
@Martinsos what about now